content/adventures/en.yaml

```diff
@@ -859,7 +859,7 @@ adventures:
                     ```
```

Comment:
Hmmm these changes are going to upset weblate probably, but also... why are they in this PR? These seem uprelated? I don't know how they got here, do you understand @hasan-sh?

Reply:
(maybe a merge from main when the conflicts are addressed will solve this?)